components/calendar/test/calendar.visual-diff.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../typography/typography.js';
+		import '../calendar.js';
+
+		const oldDate = Date;
+
+		// eslint-disable-next-line no-global-assign
+		Date = function(input1, input2, input3) {
+			if (!input1) return new oldDate('2018-02-12T12:00Z');
+			return (input2 !== 0 && !input2) ? new oldDate(input1) : new oldDate(input1, input2, input3);
+		};
+		Date.prototype = oldDate.prototype;
```

Comment:
Are you overriding `Date` so that you can control when "now" is? I think [Sinon useFakeTimers](https://sinonjs.org/releases/v1.17.6/fake-timers/) lets you do that.